### PR TITLE
test(balancer): migrate balancer test to ginkgo implement

### PR DESCRIPTION
**Pull request by TakumiInui5432**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**

Migrate balancer test to ginkgo implement.

**Related issues**
Part of #2197 
fix/resolve #0001

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [x] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [x] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

**Comment by zaunist** [comment]
cc @starsz @nic-chen

**Review by nic-chen — APPROVED** [review]

**Comment by zaunist** [comment]
cc @bzp2010

**Comment by zaunist** [comment]
@okaybase

File: api/test/e2enew/balancer/balancer_test.go [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+			}`,
+			Headers:      map[string]string{"Authorization": base.GetToken()},
+			ExpectStatus: http.StatusOK,
+			Sleep:        base.SleepTime,
+		}),
+	)
+	ginkgo.It("verify balancer by access count(same weight)", func() {
+		time.Sleep(base.SleepTime)
+		// batch test /server_port api
+		res := base.BatchTestServerPort(18, nil, "")
```
Comment (1502shivam-singh) [inline api/test/e2enew/balancer/balancer_test.go:77]: Are we like going to add more tests to extend, in the future (I pretty sure we are, but asking for validation) ? Asking this as the `headers` & `queryString` parameter of `BatchTestServerPort` are always the same here but still a function parameter.

**Review by 1502shivam-singh — APPROVED** [review]

File: api/test/e2enew/balancer/balancer_test.go (hunk 1 above)
Comment (zaunist) [reply api/test/e2enew/balancer/balancer_test.go:77]: The answer is yes, we will add more test case in the future. However, these two parameters are not currently available for testing in the dashboard, and will probably be used in the tests of the new feature.

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**